cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,68 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
+  let user;
+  
   before(() => {
-
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
 
-  it('', () => {
-
+  it('should provite the ability to register', () => {
+    cy.get('#firstName')
+      .type(user.firstName);
+    cy.gey('#lastName')
```

Comment:
There's a typo in the Cypress command. It should be `cy.get('#lastName')` instead of `cy.gey('#lastName')`.

Reply:
Done